### PR TITLE
Create error page NETSDK1141

### PR DESCRIPTION
**Pull request by ghogen**
## Summary

Add page for error message NETSDK1141 which deals with the SDK version in global.json.

Helps address issue https://github.com/dotnet/docs/issues/20656

This is a top F1 hit now that other NETSDK errors have been documented.

### CODE REVIEW

**Comment by ghogen** [comment]
@tdykstra This is a draft now from Internet research, and needs validation and review from engineering.

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:11]: This error looks much more recent than 2.1. It was added in https://github.com/dotnet/sdk/pull/12633.

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:13]: Per guidelines

```suggestion
There was a problem with the version of the SDK specified in the *global.json* file.
```

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:21]:
```suggestion
* The SDK version in the *global.json* file is incorrectly specified.
* The SDK version specified in the *global.json* file was not installed.
* The SDK version specified in *global.json* could not be found, due to an incorrect path.
```

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:17]:
```suggestion
## Cause
```

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:23]:
```suggestion
## How to fix the error
```

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:28]:
```suggestion
* Install the SDK version requested in *global.json*.
* Specify a different SDK version in *global.json*.
* Check for typos or other problems in *global.json*. See [global.json](../global-json.md) for the correct structure of that file.
* Delete *global.json*. In this case, the latest installed version of the SDK is used.
```

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:30]:
```suggestion
When you're working on a shared project, developers need to agree on the SDK version that will be used for the project. Without *global.json*, if developers on different dev machines don't have the same SDK versions, the build environment might be inconsistent across the dev team. To solve this, the SDK version can be specified in *global.json* and checked into source control as a common file which would be the same for all developers and ensure that the same SDK version is being used in all development environments. Therefore, to resolve this issue in a shared project, you might need to agree as a team on a particular SDK version and update all the code to use this version.
```

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (Youssef1313) [inline docs/core/tools/sdk-errors/netsdk1141.md:35]:
```suggestion
[How to check that the .NET SDK is installed](../../install/how-to-detect-installed-versions.md#check-sdk-versions)
```

**Comment by tdykstra** [comment]
> @tdykstra This is a draft now from Internet research, and needs validation and review from engineering.

@wli3 Who should review this PR?

File: docs/core/tools/sdk-errors/netsdk1141.md
Comment (ghogen) [reply docs/core/tools/sdk-errors/netsdk1141.md:11]: @Youssef1313 @tdykstra OK, so it was merged August 2020. Does that mean it was only added for .NET 5 ?

Comment (tdykstra) [reply docs/core/tools/sdk-errors/netsdk1141.md:11]: The [issue](https://github.com/dotnet/sdk/issues/11238) says it's for the 5.0.1xx milestone, so I would change the Applies-to line, though I think these Applies-to lines on error messages are superfluous -- if you get the error message and look it up, it definitely applies to you.

```suggestion
**This article applies to:** ✔️ .NET Core 5.0 SDK and later versions
```

**Comment by ghogen** [comment]
@Youssef1313 @tdykstra Ready to merge?

**Review by tdykstra — APPROVED** [review]
Well done -- comprehensive, clear, and concise.

**Pull request merged**
